platform/viewer/src/context/SnackbarContext.js

```diff
@@ -0,0 +1,102 @@
+import React, { useState, createContext, useContext } from 'react';
+import SnackbarContainer from '../../../ui/src/components/snackbar/SnackbarContainer';
+import SnackbarTypes from '../../../ui/src/components/snackbar/SnackbarTypes';
```

Comment:
Similarly, these will work okay in our PWA build, but it will cause our `@ohif/ui` and `@ohif/viewer` packages to fail.